include/numerics/numeric_vector.h

```diff
@@ -191,36 +197,36 @@ class NumericVector : public ReferenceCountedObject<NumericVector<T>>,
   virtual std::unique_ptr<NumericVector<T>> clone () const = 0;
 
   /**
-   * Change the dimension of the vector to \p N. The reserved memory
-   * for this vector remains unchanged if possible.  If \p N==0, all
+   * Change the dimension of the vector to \p n. The reserved memory
+   * for this vector remains unchanged if possible.  If \p n==0, all
    * memory is freed. Therefore, if you want to resize the vector and
    * release the memory not needed, you have to first call \p init(0)
-   * and then \p init(N). This behaviour is analogous to that of the
+   * and then \p init(n). This behaviour is analogous to that of the
    * STL containers.
    *
    * On \p fast==false, the vector is filled by zeros.
    */
-  virtual void init (const numeric_index_type,
-                     const numeric_index_type,
-                     const bool = false,
-                     const ParallelType = AUTOMATIC) = 0;
+  virtual void init (const numeric_index_type n,
+                     const numeric_index_type n_local,
+                     const bool fast = false,
+                     const ParallelType ptype = AUTOMATIC) = 0;
 
   /**
    * Call \p init() with n_local = N.
    */
-  virtual void init (const numeric_index_type,
-                     const bool = false,
-                     const ParallelType = AUTOMATIC) = 0;
+  virtual void init (const numeric_index_type n,
+                     const bool fast = false,
+                     const ParallelType ptype = AUTOMATIC) = 0;
 
   /**
    * Create a vector that holds tha local indices plus those specified
    * in the \p ghost argument.
    */
-  virtual void init (const numeric_index_type /*N*/,
-                     const numeric_index_type /*n_local*/,
-                     const std::vector<numeric_index_type> & /*ghost*/,
-                     const bool /*fast*/ = false,
-                     const ParallelType = AUTOMATIC) = 0;
+  virtual void init (const numeric_index_type n,
+                     const numeric_index_type n_local,
+                     const std::vector<numeric_index_type> & ghost,
+                     const bool fast = false,
+                     const ParallelType ptype = AUTOMATIC) = 0;
```

Comment:
Yeah, this looks like support for my "overzealous compiler" theory, in which case we need to keep the comment wrapping in.

Reply:
The problem is that the documentation is referring to these variable names.

Reply:
No, after digging through our svn history, it looks like my hypothesis is just wrong.

The comment wrapping wasn't here because there was a compiler that could warn about unused variables in declarations, the comment wrapping was here because it predates when we split the definitions apart from the declarations.  So it's now just an atavism and you're right to remove it.

Reply:
That makes sense.